projects/PackableTestProject/PackableTestProject.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This and the implicit project are both for Packable and Publishable. Can you either separate the two or make it clear in the project name.

Reply:
Suggestion for the name?

Reply:
PackableAndPublishableTestProject

Reply:
Or without the 'And'